bowlingGame.js

```diff
@@ -0,0 +1,68 @@
+class BowlingGame {
+  constructor(rollList = []) {
+    this.rollList = rollList;
+    this.sum = 0;
```

Comment:
As mentioned previously, I'd work to remove this.
A good way to think about which variables you should store is that information should not be duplicated inside instance variables. Here the rollList is enough to fully describe the state of your system (including the score, potential strikes, etc), so it really should be the only instance variable of the class.